source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,76 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
+ - Fixed an issue with extra space getting added to code blocks in search results.
+ - Fixed an issue where deactivated members were not included in a favorited **Direct Message** channel export.
+ - Fixed an issue where password strength settings wouldn't be disabled if they were set through environment variables.
+ - Fixed an issue where post mentions would grow outside the viewport on small devices.
+ - Fixed an issue with draft removal after deleting the post.
+ - Fixed a markdown issue where, on some occasions, extra space was found before a list.
+ - Fixed an issue where a sender to a custom group would also receive the message notification themselves.
+ - Fixed a web app crash when a System Admin clicked on a link to a private channel that they were not a member of.
+ - Fixed ``ChannelHasBeenCreated`` plugin hook not being called when a group channel was created.
+ - Fixed thread notifications so that if a user had **Thread Reply Notifications** disabled for your account and **Automatically follow threads in this channel** enabled for a channel, the user wouldn't receive thread notifications for that channel per global setting.
+
+### config.json
+ - Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to the Enterprise plan:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added ``EnableOutgoingOAuthConnections`` configuration setting for Outgoing OAuth Connections integration type.
+
+### Open Source Components
+ - Added ``@floating-ui/react``, and removed ``@floating-ui/react-dom`` and ``@floating-ui/react-dom-interactions`` from https://github.com/mattermost/mattermost/.
+
+### Go Version
+ - v9.6 is built with Go ``v1.20.7``.
+
+### Known Issues
+ - Draft is lost from the reply box when other user's root post is deleted [MM-56972](https://mattermost.atlassian.net/browse/MM-56972).
+ - Users' initial status is not always loaded correctly [MM-56966](https://mattermost.atlassian.net/browse/MM-56966).
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
+ - The Playbooks left-hand sidebar doesn't update when a user is added to a run or playbook without a refresh.
+ - If a user isn't a member of a configured broadcast channel, posting a status update might fail without any error feedback. As a temporary workaround, join the configured broadcast channels, or remove those channels from the run configuration.
+ 
+### Contributors
+ - 
+
```

Comment:
Not sure if this is a placeholder but the section is currently blank

Reply:
This is a placeholder for now until we get the list on March 1st.